LookingMenuRX/Scenes/App/AppViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import RxSwift
+import RxCocoa
+import MGArchitecture
+
+struct AppViewModel {
+    let navigator: AppNavigatorType
+}
+
+extension AppViewModel: ViewModel {
+    
+    struct Input {
+        let loadTrigger: Driver<Void>
+    }
+    
+    struct Output {
+        let toMain: Driver<Void>
+    }
+    
+    func transform(_ input: AppViewModel.Input, disposeBag: DisposeBag) -> AppViewModel.Output {
+        
+        let toMain = input.loadTrigger
+            .do(onNext: { _ in
+                navigator.toMain()
+            })
```

Comment:
```suggestion
            .do(onNext: navigator.toMain)
```
Em có thể dùng như này nhé. Không nhất thiết phải sửa